apps/src/gamelab/animationListModule.js

```diff
@@ -229,11 +229,15 @@ export function isNameUnique(name, animationListProps) {
 function generateAnimationName(baseName, animationList) {
   let unavailableNumbers = [];
   // Match names with the form baseName_#
-  const re = new RegExp(`^${baseName}_(\\d+)$`);
   for (let animation in animationList) {
-    let match = re.exec(animationList[animation].name);
-    if (match !== null) {
-      unavailableNumbers.push(parseInt(match[1]));
+    let animationName = animationList[animation].name;
+    if (animationName.startsWith(baseName)) {
```

Comment:
I would validate that this works in IE (which doesn't support startsWith natively). It's possible we'll need a polyfill.